demo/drag-and-drop.html

```diff
@@ -0,0 +1,341 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+<title>three-pathfinding demo</title>
+<link href="style.css" rel="stylesheet">
+<body>
+
+	<demo-sidebar>
+		<p>
+			Drag and drop gltf/glb files or choose a file here:
+		</p>
+		<input type="file" id="load-file"/>
+		<p>
+			If you need example glb files you can find some here:
+		</p>
+		<ul>
+			<li><a href="/drag-and-drop/tower-small.glb">tower-small.glb</a></li>
+			<li><a href="/drag-and-drop/tower-middle.glb">tower-middle.glb</a></li>
+			<li><a href="/drag-and-drop/tower-big.glb">tower-big.glb</a></li>
+		</ul>
+		These examples has been created with <a href="https://www.blender.org/">Blender.</a>
+		If you need an example how a navmesh should look like in a blender file
+		you can check out these files:
+		<ul>
+			<li><a href="/drag-and-drop/tower-small.blend">tower-small.blend</a></li>
+			<li><a href="/drag-and-drop/tower-middle.blend">tower-middle.blend</a></li>
+			<li><a href="/drag-and-drop/tower-big.blend">tower-big.blend</a></li>
```

Comment:
Minor — Let's move the .blend and .glb files to the `assets/` folder, at least in theory we could use them in other examples, and that way static assets are in a single place. Possibly worth including a [`download` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) as well.